fitparser/src/de/parser.rs

```diff
@@ -225,97 +225,41 @@ impl FitDataMessage {
         self.time_offset
     }
 
-    /// Data field mapping of <field_number, Value>
-    pub fn fields(&self) -> &HashMap<u8, Value> {
+    /// Data field mapping of <(dev_data_idx, field_number), Value>
+    pub fn fields(&self) -> &HashMap<(u8, u8), Value> {
         &self.fields
     }
 
-    /// Mutable Data field mapping of <field_number, Value>
-    pub fn fields_mut(&mut self) -> &mut HashMap<u8, Value> {
+    /// Mutable Data field mapping of <(dev_data_idx, field_number), Value>
+    pub fn fields_mut(&mut self) -> &mut HashMap<(u8, u8), Value> {
         &mut self.fields
     }
-
-    #[allow(dead_code)]
-    /// Developer field data
-    pub fn developer_fields(&self) -> &[Value] {
-        &self.developer_fields
-    }
-}
-
-/// Base types defined by the FIT protocol. The "z" variants have a different invalid value
-/// than the versions without the suffix.
-#[derive(Clone, Copy, Debug)]
-enum BaseType {
```

Comment:
I don't remember anymore why I didn't use the one defined in the profile but lets roll with it until we find a problem!